Update changes-after-migration.md (#2378)
SumoLogic/sumologic-documentation: docs/dashboards-new/dashboards-migration/changes-after-migration.md

## Color Mapping

> In the Classic dashboard, there were 6 color schemes, but there are 11 in the New dashboard. The color mapping from the Classic to the New dashboard is shown in the image below.

Reviewer: ```suggestion
Dashboards (Classic) had 6 color schemes, while Dashboards (New) have 11 color schemes. The color mapping from the Classic to the New dashboard is shown in the image below.
```
